code/modules/reagents/chemistry/reagents/medicine.dm

```diff
@@ -1289,11 +1294,28 @@
 				return ..()
 			else
 				for(var/obj/item/organ/external/E in M.bodyparts)
-					if(E.is_broken())
-						if(prob(50)) // Each tick has a 50% chance of repearing a bone.
+					if(prob(25)) // Each tick has a 50% chance of repearing a bone.
+						if(E.status & (ORGAN_INT_BLEEDING | ORGAN_BROKEN | ORGAN_SPLINTED | ORGAN_DEAD)) //I can't just check for !E.status
```

Comment:
I'm not sure `ORGAN_DEAD` should be included here, if it's completely necrosed I'm not sure it should be repairable